csharp-dotnet/README.md

```diff
@@ -0,0 +1,110 @@
+# **C#/.NET Programming Standards**
+
+## **Introduction**
+
+This guide has been written to provide AMU programmers with a set of coding standards and conventions to follow when developing new .NET applications. These guidelines borrow from Microsoft's  [C# Coding Conventions](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/inside-a-program/coding-conventions).
+
+These .NET programming standards and best practices promote a higher level of maintainability and readability within the software.
+
+AMU benefits:
+
+- Create a consistent look to the code, so that readers can focus on content, not layout.
+- Create a common reference point for development and technical discussions.
+- Reduce learning curve for new developers.
+- Demonstrate C# best practices and reduce common coding issues.
+
+
+
+## Layout Conventions
+
+Microsoft examples and samples conform to the following conventions:
+
+- Use the default Code Editor settings (smart indenting, four-character indents, tabs saved as spaces). For more information, see [Options, Text Editor, C#, Formatting](https://docs.microsoft.com/en-us/visualstudio/ide/reference/options-text-editor-csharp-formatting).
```

Comment:
I don't personally mind 4 character indents for c#, but it's worth noting this would be a departure from the 2 space rule we've used in the past (at least in the Digital Products side of things).

Reply:
Noted in document - C# only